Fixes error when compiling Java implementation with recent OpenJDK:

    error: Source option 7 is no longer supported. Use 8 or later.
    error: Target option 7 is no longer supported. Use 8 or later.

--- a/Rakefile
+++ b/Rakefile
@@ -13,6 +13,8 @@
 Rake::JavaExtensionTask.new('concurrent_ruby', core_gemspec) do |ext|
   ext.ext_dir = 'ext/concurrent-ruby'
   ext.lib_dir = 'lib/concurrent-ruby/concurrent'
+  ext.source_version = '8'
+  ext.target_version = '8'
 end
 
 unless Concurrent.on_jruby? || Concurrent.on_truffleruby?
